api/types/events/events.proto

```diff
@@ -1025,6 +1025,9 @@ message SFTP {
 
     // Action is what kind of file operation
     SFTPAction Action = 11 [ (gogoproto.jsontag) = "action" ];
+
+    // Error is the optional error that may have occurred
+    string Error = 12 [ (gogoproto.jsontag) = "error,omitempty" ];
```

Comment:
I don't think this change is related to GitHub

Reply:
Oops, guess I accidentally included a commit from my other branch somehow